src/nvim/tui/tui.c

```diff
@@ -208,10 +210,12 @@ static void try_resize(Event ev)
   UI *ui = ev.data;
   update_size(ui);
   ui_refresh();
+  during_winch = false;
 }
 
 static void sigwinch_cb(SignalWatcher *watcher, int signum, void *data)
 {
+  during_winch = true;
```

Comment:
`try_resize` is only called by `sigwinch_cb`. So we don't need to track state at all--we can add a parameter to  `tui_resize` called `try_resize_terminal` and check that before sending the terminal resize sequence.